bootloader/src/nRF52840/nrf_it.c

```diff
@@ -83,7 +83,7 @@ void HardFault_Handler(void)
         " mrseq r0, msp                                             \n"
         " mrsne r0, psp                                             \n"
         " ldr r1, [r0, #24]                                         \n"
-        " ldr r2, handler2_address_const                            \n"
+        " ldr r2, =handler2_address_const                           \n"
```

Comment:
Changing this to a pseudo-instruction fixes warnings that are now generated by the assembler bundled with ARM GCC 10:
```
/tmp/ccfCrEXn.s: Assembler messages:
/tmp/ccfCrEXn.s:65974: Warning: section does not have enough alignment to ensure safe PC-relative loads
```